cli/commands/benchmark-web-vitals.mjs

```diff
@@ -449,12 +468,15 @@ async function benchmarkURL(
 			await page.emulateNetworkConditions( params.networkConditions );
 		}
 
-		await page.setViewport( params.windowViewport );
-		await page
-			.mainFrame()
-			.waitForFunction(
-				`window.innerWidth === ${ params.windowViewport.width } && window.innerHeight === ${ params.windowViewport.height }`
-			);
```

Comment:
I removed this code because it with the `deviceScaleFactor` coming from device emulation, I found the `innerWidth` and `innerHeight` to be the viewport's dimensions multiplied by the device scale factor. While I could implement that adjustment, I didn't find that this code was actually necessary.